### DISCUSSION TITLE
SwiftSyntax parser library isn't compatible

### DISCUSSION THREAD

**Discussion by askiefer** [vinivendra/Gryphon#97] [Q&A] [answered]
Hello, I just installed `gryphon` and am trying to follow the single file simple tutorial. 

When I run gryphon main.swift I receive the error:
```
SwiftSyntax parser library isn't compatible
```
Any ideas? I am on a Mac using Swift 5.3, Xcode 11.3.1

**Comment by vinivendra** [comment, marked as answer]
Hello and welcome :)

Yeah, SwiftSyntax can be a bit annoying to deal with sometimes. I have a feeling you might be able to fix this by updating Xcode, but I'm not sure.

First, I need to know your versions exactly. Can you run the `xcode-select -p` and `swift --version` commands and tell me exactly what they print? And what version of macOS are you using?

Also, did you by any chance install Gryphon with a different Swift/Xcode version than what you're using to run it?

**Reply by askiefer** [reply]
Happy to, thanks. When I run `xcode-select -p` it prints:
```
/Applications/Xcode.app/Contents/Developer
```
The second command prints:
```
Apple Swift version 5.1.3 (swiftlang-1100.0.282.1 clang-1100.0.33.15)
Target: x86_64-apple-darwin18.7.0
```
I'm on MacOS Mojave Version 10.14.6 (18G103)


Hm not sure, I built gryphon from source by git cloning the repo and running the install script.

I tried to install it using brew first and received this:
```
==> swift build --disable-sandbox
Last 15 lines from /Users/annas.kiefer/Library/Logs/Homebrew/gryphon/02.swift:
2020-12-22 09:35:21 -0800

swift
build
--disable-sandbox

/private/tmp/gryphon-20201222-7413-lcqnkl/Gryphon-0.10.6: error: manifest parse error(s):
<unknown>:0: error: Swift does not support the SDK 'MacOSX10.14.sdk'
```

**Reply by vinivendra** [reply]
Oh, I think that's the problem: from your output of `swift --version` it seems like you're using Swift 5.1.3, not 5.3. Swift 5.1.3 was supported by earlier versions of Gryphon, but not anymore.

I'd recommend you either update Xcode (to version [11.4 or higher](https://xcodereleases.com)) or download [Swift 5.2 or 5.3](https://swift.org/download/) directly.

If that doesn't work, let me know!

**Reply by askiefer** [reply]
Ah okay awesome, thank you, I will update Xcode and try again!
